src/style.css

```diff
@@ -1 +1,63 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo__box {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__panel {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+  position: relative;
```

Comment:
Please pay attention to your css properties order
https://9elements.com/css-rule-order/

Reply:
Thanks for the info! I have made corrections according to it.